src/LeanCode.Contracts/ITopic.cs

```diff
@@ -0,0 +1,7 @@
+namespace LeanCode.Contracts;
+
+/// <summary>
+/// Marker interface, do not use directly.
```

Comment:
Is this correct? You do seem to be using them directly in examples.

Reply:
It's not, I'll remove these comments.